### PR TITLE
Bug 1536963 Updates required to work with openshift 0.4.0

### PR DESCRIPTION
**Pull request by jmontleon**
We're going to need to wait for this to be available at the least.
<fabianvf> jmontleon: just tagged v0.4.0.a1 which should push itself to pypi once the build passes

It's also entirely possible I've missed other places that need updating.

### CODE REVIEW

**Review by fabianvf — APPROVED** [review]
Once 0.4.0.a1 is pushed to pypi I think this should all work fine.

**Review by djzager — APPROVED** [review]
You probably need a bugzilla for this.

**Comment by djzager** [comment]
@jmontleon I think this is the bug you should use https://bugzilla.redhat.com/show_bug.cgi?id=1536963

**Review by karmab — APPROVED** [review]

File: src/apb/engine.py [hunk 1]
```diff
@@ -433,8 +433,8 @@ def broker_resource_url(host, broker_name):
 def relist_service_broker(kwargs):
     try:
         openshift_config.load_kube_config()
-        token = openshift_client.configuration.api_key['authorization']
-        cluster_host = openshift_client.configuration.host
+        token = openshift_client.Configuration().get_api_key_with_prefix('authorization')
```
Comment (djzager) [inline src/apb/engine.py:436]: This also exists on line 1092.

**Review by djzager — COMMENTED** [review]
Looks like one more spot.

**Review by cfchase — APPROVED** [review]

File: src/apb/engine.py (hunk 1 above)
Comment (jmontleon) [reply src/apb/engine.py:436]: Thanks. Updated the PR message and fixed 1092.

**Review by dymurray — APPROVED** [review]

**Review by dymurray — COMMENTED** [review]
Looks like installing openshift 0.4.0 is failing on Travis. The latest version in pypi is 0.3.4.

**Comment by djzager** [comment]
"<fabianvf> jmontleon, karimb, dzager: openshift 0.4.0.a1 is now up on pypi, pip install 'openshift==0.4.0.a1' should grab it"

I see there is https://pip.pypa.io/en/stable/reference/pip_install/#pre-release-versions for installing pre-release versions. Not sure if there is a way to use that so we don't lock it down to the alpha version forever.

**Pull request merged**